source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputToggleRenderer.mm

```diff
@@ -37,38 +38,47 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 {
     std::shared_ptr<HostConfig> config = [acoConfig getHostConfig];
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
-    std::shared_ptr<ToggleInput> toggleBlck = std::dynamic_pointer_cast<ToggleInput>(elem);
+    std::shared_ptr<ToggleInput> adaptiveToggleInput = std::dynamic_pointer_cast<ToggleInput>(elem);
 
     NSBundle *bundle = [NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"];
     if (!bundle) { // https://github.com/Microsoft/AdaptiveCards/issues/1834
         return nil;
     }
-    ACRInputTableView *inputTableView = [bundle loadNibNamed:@"ACRInputTableView" owner:self options:nil][0];
-    inputTableView.frame = CGRectMake(0, 0, viewGroup.frame.size.width, viewGroup.frame.size.height);
-    [inputTableView setSeparatorStyle:UITableViewCellSeparatorStyleNone];
-    ACRToggleInputDataSource *dataSource = [[ACRToggleInputDataSource alloc] initWithInputToggle:toggleBlck WithHostConfig:config];
-    dataSource.parentStyle = viewGroup.style;
-    inputTableView.delegate = dataSource;
-    inputTableView.dataSource = dataSource;
 
+    ACRToggleInputView *toggleView = [[ACRToggleInputView alloc] initWithFrame:CGRectMake(0, 0, viewGroup.frame.size.width, viewGroup.frame.size.height)];
+
+    toggleView.title.text = [NSString stringWithCString:adaptiveToggleInput->GetTitle().c_str() encoding:NSUTF8StringEncoding];
+    toggleView.title.textColor = getForegroundUIColorFromAdaptiveAttribute(config, viewGroup.style);
+    toggleView.title.adjustsFontSizeToFitWidth = NO;
+    if (!adaptiveToggleInput->GetWrap()) {
+        toggleView.title.numberOfLines = 1;
```

Comment:
>1; [](start = 41, length = 2)

In general for code readability avoid hardcoded literals and magic numbers; Trying to understand the logic here - So if wrap property isnt specified we interpret/default to 1 for its title line count? This almost feels like an initialization operation that say can be handled at the constructor for ACRToggleInputView no?

Reply:
iOS renderer defaults number of lines to zero. If zero, text box will keep increasing its height with longer text. iOS renderer limits number of lines to 1, and its effect is same as having wrap property set to "false".